com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -242,7 +242,11 @@ public int getMaxCpuThreads() {
         if (maxThreadsOpt == null) {
             return getDefaultMaxCpuThreads();
         }
-        return Integer.parseInt(maxThreadsOpt);
+        int threads = Integer.parseInt(maxThreadsOpt);
+        if (threads == 0) {
+            threads = Runtime.getRuntime().availableProcessors();
```

Comment:
I talked with a guy who works on CI builds a lot, and he said it might be a problem if we use all the threads because it may be a reason of timeout issues if runner couldn't answer the isLive request. Not sure if it might be a problem in our case and if we should use `-1` here.

Reply:
Ok, reducing by one!